libs/partners/mongodb/langchain_mongodb/vectorstores.py

```diff
@@ -183,6 +183,8 @@ def _similarity_search_with_score(
         k: int = 4,
         pre_filter: Optional[Dict] = None,
         post_filter_pipeline: Optional[List[Dict]] = None,
+        remove_embedding: bool = True,
```

Comment:
I usually find it unnatural to have a optional argument that defaults to True (since usually missing is false). Could we name this "include_embedding" and default to False instead?